### PR TITLE
chore: release 0.28.0

### PR DESCRIPTION
**Pull request by Fabien-GELUS**
Signed-off-by: Fabien Gelus <fabien.gelus@owkin.com>

## Related issue

`#` followed by the number of the issue

## Please check if the PR fulfills these requirements

- [ ] If the feature has an impact on the user experience, the [changelog](https://github.com/substra/substrafl/blob/main/CHANGELOG.md) has been updated
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The commit message follows the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification

### CODE REVIEW

**Review by cboillet-dev — APPROVED** [review]

**Comment by Owlfred** [comment]
```
Usage: /e2e [options] [help]

/e2e may appear anywhere as long as it is on its own line

Options:
  --refs <value>                                   Extra refs (branch or tag) with format REPO=GIT_REF,REPO=GIT_REF.
  Supported repositories: substra-hlf-k8s, orchestrator, substra-backend, substra-frontend, substra-tools, substrafl, substra, substra-tests, connect-ci.
  Example: /e2e --refs substra-backend=some_branch,orchestrator=some_tag (default: {})
  --tests-to-run, --tests <tests-to-run>           Comma-separated list of tests to run. Valid options: sdk,workflow,substrafl,frontend or NONE. (default: "sdk")
  --orchestrator-mode, --mode <orchestrator-mode>  Comma-separated list of orchestrator modes to run tests for. Valid options: standalone,distributed (default: "standalone,distributed")
  -h, --help                                       display help for command
```

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by Fabien-GELUS** [comment]
/e2e  --tests-to-run substrafl --orchestrator-mode standalone

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

File: substrafl/remote/register/register.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 logger = logging.getLogger(__name__)
 
 # Substra tools version for which the image naming scheme changed
-MINIMAL_DOCKER_SUBSTRATOOLS_VERSION = "0.10.0"
+MINIMAL_DOCKER_SUBSTRATOOLS_VERSION = "0.16.0"
```
Comment (Esadruhn) [inline substrafl/remote/register/register.py:30]: do not change this line, the `MINIMAL_DOCKER_SUBSTRATOOLS_VERSION` is relative to the naming scheme

Comment (Fabien-GELUS) [reply substrafl/remote/register/register.py:30]: but the previous images do not exist on ghcr.io/substra/substra-tools ?

Comment (Esadruhn) [reply substrafl/remote/register/register.py:30]: ah true, should have been changed when we changed the image name

**Review by Esadruhn — APPROVED** [review]

**Pull request merged**